backends/platform/ios7/ios7_options.mm

```diff
@@ -123,6 +132,12 @@
 	        .addLayout(GUI::ThemeLayout::kLayoutVertical)
 	            .addPadding(0, 0, 0, 0)
 	            .addWidget("OnScreenControl", "Checkbox")
+	            .addWidget("OnScreenControlOpacity", "OptionsLabel")
+	        .addLayout(GUI::ThemeLayout::kLayoutHorizontal)
```

Comment:
Is there a reason not to include the OnScreenControlOpacity into the horizontal layout?

Reply:
The text became cropped with ...
But maybe I did it wrong?

Reply:
Ah, I changed transparency to opacity so now it will actually fit. Pushed an update